lib/css/components/_stacks-topbar.less

```diff
@@ -0,0 +1,366 @@
+                position: absolute;
+                content: '';
+                width: 18px;
+                height: 2px;
+                left: 0;
+                background: var(--theme-topbar-item-color);
+                top: -5px;
+                transition: all ease-in-out 0.1s;
+            }
+
+            &:after {
+                top: 5px;
+            }
+        }
+
+        // Transforming hamburger into x
+        &.is-selected {
+            color: var(--theme-topbar-item-color-current);
+            background-color: var(--theme-topbar-item-background-current);
+
+            span {
+                background-color: transparent;
+
+                &:before,
+                &:after {
+                    top: 0;
+                    transform: rotate(-45deg);
+                }
+
+                &:after { transform: rotate(45deg); }
+            }
+        }
+
+        &:hover {
+            color: var(--theme-topbar-item-color-hover);
+            background-color: var(--theme-topbar-item-background-hover);
+        }
+    }
+}
+
+.s-topbar .s-topbar--navigation,
+.s-topbar .s-topbar--content,
+.s-topbar .s-topbar--ctas {
+    display: flex;
+    height: 100%;
+    .list-reset;
+
+    // TODO cleanup
+    > li {
+        display: inline-flex;
+    }
+}
+
+.s-topbar .s-topbar--navigation,
+.s-topbar .s-topbar--content {
+    .s-topbar--item {
+        color: var(--theme-topbar-item-color);
+        display: inline-flex;
+        align-items: center;
+        padding: 0 10px; // TODO @su8 or @su12?
+        text-decoration: none;
+        white-space: nowrap;
+        position: relative;
+        white-space: nowrap;
+
+        &:hover, &:focus,
+        &.is-selected, &.is-selected:hover, &.is-selected:focus {
+            color: var(--theme-topbar-item-color-hover);
+            background-color: var(--theme-topbar-item-background-hover);
+            text-decoration: none;
+            outline: none;
+
+            .s-activity-indicator {
+                top: 6px;
+            }
+        }
+
+        .svg-icon {
+            vertical-align: text-top;
+        }
+
+        .s-activity-indicator {
+            position: absolute;
+            top: 10px;
+            right: 2px;
+            transition: top @te-smooth 0.15s;
+        }
+    }
+}
+
+// Primary navigation: Links to products on the left
+.s-topbar .s-topbar--navigation {
+    .s-topbar--item {
+        display: inline-block;
+        padding: @su4 + @su2 @su16;
+        border-radius: 1000px;
+        align-self: center;
+
+        &.is-selected {
+            background-color: var(--theme-primary-color);
+            color: var(--white);
+
+            &:hover,
+            &:focus {
+                background-color: var(--theme-primary-600);
+                color: var(--white);
+            }
+        }
+    }
+
+    // TODO Stacks component? Badge?
+    .s-topbar--item.s-topbar--notice {
+        border-radius: @br-sm;
+        font-weight: bold;
```

Comment:
We have more control these days. `700` or `800` ought to do it.

Reply:
Do you have any specific tips on when to use `bold` vs the numeric values? I don't personally have any specific sense for when one looks better than another or on what fonts these additional values are supported. Fwiw, it seems like Less is compiling this into `700`. I didn't realize it did that!